src/app/LoginPanel.tsx

```diff
@@ -39,28 +39,30 @@ export default function LoginPanel() {
       dispatch(OrgActions.set({ mine: res.organization }));
     } else {
       const organization = res.organization;
-      let supportEmail: string | undefined = undefined;
+      let supportContact: string | undefined = undefined;
       let memberProviderName = undefined;
 
       if (organization) {
-        supportEmail = organization.supportEmail;
+        supportContact = organization.supportEmail;
         memberProviderName = MemberProviderName[organization.memberProvider];
       }
 
+      supportContact ??= "your unit's operations leader";

```

Comment:
Operations Leader seems too specific to our org structure. Maybe something more generic like system administrator?

Reply:
Operations Leader was suggested somewhere in Slack before. The problem with System Administrator is I don't think anyone would know who that is unless they're already involved in the tech side of things.

When you say "our org structure", who are you referring to? I know that applies to 4x4 and ESAR. Doesn't it apply to all units in KCSARA?

Reply:
I just don't think an operations leader is the right person to contact, and if we go outside of King county other organizations may not have that title or role

Reply:
WASART is a possible example that I could see us supporting in the nearer term. They do not have "OLs".

Reply:
Ahh and of course in my head I'm thinking "leader of operations" administratively...i.e. VPO or Operations Chair. But there's not really a standard term for that either.

I guess I can use System Administrator and just trust the user to escalate up their chain of command appropriately. Will make that change.

Reply:
Done.